libs/sdk-bindings/makefile

```diff
@@ -30,19 +30,30 @@ darwin-universal: $(SOURCES)
 	lipo -create -output ../target/darwin-universal/release/libbreez_sdk_bindings.dylib ../target/aarch64-apple-darwin/release/libbreez_sdk_bindings.dylib ../target/x86_64-apple-darwin/release/libbreez_sdk_bindings.dylib
 	lipo -create -output ../target/darwin-universal/release/libbreez_sdk_bindings.a ../target/aarch64-apple-darwin/release/libbreez_sdk_bindings.a ../target/x86_64-apple-darwin/release/libbreez_sdk_bindings.a
 
+csharp-darwin: darwin-universal
```

Comment:
Should we add `csharp-darwin` & `csharp-linux` targets to `make all`?

Reply:
Sounds like it should belong there.